AGENTS.md

```
@@ -0,0 +1,215 @@
+
+### Build
+
+```bash
+python -m build -o dist/
+```
+
+## Architecture
+
+### Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)